docs/development/build-instructions-gn.md

```diff
@@ -328,3 +328,20 @@ We recommend installing Node through [nvm](https://github.com/nvm-sh/nvm). This
 ### RBE authentication randomly fails with "Token not valid"
 
 This could be caused by the local clock time on the machine being off by a small amount. Use [time.is](https://time.is/) to check.
+
+
+### Memory Integrity (MTE) on macOS arm64
+
+Electron enables Clang's hardware memory tagging on Apple Silicon by default.
+
+- Enabled when: is_mac && current_cpu == "arm64" and your toolchain supports it.
+- Compiler/linker flags used: -fsanitize=memtag
+- Compatibility: Requires a recent Apple Clang and macOS SDK (for example, Xcode 15+/macOS 14+). On unsupported toolchains, you can disable it.
```

Comment:
```suggestion
- Compatibility: Requires a recent Apple Clang and macOS SDK (for example, Xcode 15+/macOS 14+).
  On unsupported toolchains, you can disable it.
```

Our [docs style guide](https://www.electronjs.org/docs/latest/development/style-guide) recommends wrapping lines between 80 and 100 characters.